app/src/main/res/layout/fragment_adminpanel_stocks.xml

```diff
@@ -83,6 +83,7 @@
 
 
         <View
+            android:id="@+id/view5"
```

Comment:
remove numerical suffixes and give proper name to all views in this PR @satyamurti  !

Reply:
ok